proto/cosmos/gov/v1/gov.proto

```diff
@@ -304,15 +310,21 @@ message Params {
   // Since: cosmos-sdk 0.50
   string min_deposit_ratio = 16 [(cosmos_proto.scalar) = "cosmos.Dec"];
 
+  // proposal_cancel_max_cancel_period defines how far in the voting period a proposer can cancel a proposal.
+  // If the proposal is cancelled before the max cancel period, the deposit will be returned/burn to the
+  // depositors, according to the proposal_cancel_ratio and proposal_cancel_dest parameters.
+  // After the max cancel period, the proposal cannot be cancelled anymore.
+  string proposal_cancel_max_cancel_period = 17 [(cosmos_proto.scalar) = "cosmos.Dec"];
```

Comment:
```suggestion
  string cancel_proposal_max_period = 17 [(cosmos_proto.scalar) = "cosmos.Dec"];
```
This field name seems a bit long, you might consider short name while preserving clarity

Reply:
I chose this name to preserve consistency with the other cancel proposal params. They are written as `proposal_cancel_xxx`

Reply:
oh okay,
its not a strong suggestion but then maybe its better to use `proposal_cancel_max_period` to avoid repeating words.

Reply:
Do you find clear enough that `max_period` refers to the max cancellation period then?

Reply:
`max_period` intuitively conveys the idea of a max duration or time period, and here in the context of proposal cancellation, I believe its clear and concise

Reply:
Alright, I will update then 👍🏾